### ISSUE TITLE
PHPStan is failing

### ISSUE THREAD

**Issue by leoloso** [migrify/migrify#195]
It's throwing these errors:

```

 ------------------------------------------------------------------------
  packages/php-config-printer/tests/Printer/SmartPhpConfigPrinter/Source/ValueObject/SimpleValueObject.php:7
 ------------------------------------------------------------------------
  - '#Value Object class name "SimpleValueObject" must be withotu "ValueObject" suffix\. The correct class name is "Simple"#'
 ------------------------------------------------------------------------

     -
         message: '#Property with protected modifier is not allowed\. Use interface instead#'
         paths:
             - packages/config-transformer/tests/Converter/ConfigFormatConverter/AbstractConfigFormatConverterTest.php # 26
             - packages/config-transformer/tests/Converter/ConfigFormatConverter/AbstractConfigFormatConverterTest.php # 31
             - packages/config-transformer/tests/Converter/ConfigFormatConverter/AbstractConfigFormatConverterTest.php # 36


                                                                                
 [ERROR] Found 4 errors                                                         
                                                                                

```

**Comment by TomasVotruba** [comment]
Hi, thanks for reporting.
Could you enclose PR with fixes too?

**Comment by leoloso** [comment]
I have no problem, but I think it's not worth it: this issue has already been resolved for Rector master... so somebody already knows what the problem is and how to fix it (was it you?).

So, if I try to fix it here, it'd be a duplicated effort (and you're paying for it! 🤣 )...

I created this issue here, mostly so that you'd be aware it's happening here too...

**Comment by TomasVotruba** [comment]
That's exactly purpose of delegation. To save the effort.
Go for it, I believe you can make it 👍

**Comment by leoloso** [comment]
Fixed in #194:

- Replicated `composer: bump symplify to 8.3.35 with stan fix ` (https://github.com/rectorphp/rector/commit/15600516af42fd493cfbba54e08d72cc2ff3ec8f)
- Ignore those errors in `phpstan.neon`
